backend/docs/D4_Installing_python_packages.md

```diff
@@ -0,0 +1,28 @@
+# Installing python packages
+
+If you want to install new packages, you can do so by adding the dependencies to:
+
+* `requirements.in/base.txt` for packages that are needed in both *development* **and** *production*
+
+* `requirements.in/dev.txt` for packages that are only needed in *development*
+
+* `requirements.in/prod.txt` for packages that are only needed in *production*
+
+Then, run the following `pip-compile` commands to generate the `requirements.txt` files:
+
+```bash
+# Install all packages for development
+docker compose run --rm server pip-compile --output-file=requirements/dev.txt requirements.in/dev.txt
+
```

Comment:
This actually doesn't install the packages, but will check the `requirements.in` file, and check
- which versions are compatible with the current Python version
- which dependencies are needed for the packages listed in `requirements.in`